packages/common/src/services/integrations/jwp/JWPCheckoutService.ts

```diff
@@ -50,42 +60,24 @@ export default class JWPCheckoutService extends CheckoutService {
    * Format a (Cleeng like) offer id for the given access fee (pricing option). For JWP, we need the asset id and
    * access fee id in some cases.
    */
-  private formatOfferId(offer: AccessFee) {
-    const ppvOffers = ['ppv', 'ppv_custom'];
-
-    return ppvOffers.includes(offer.access_type.name) ? `C${offer.item_id}_${offer.id}` : `S${offer.item_id}_${offer.id}`;
-  }
-
-  /**
-   * Parse the given offer id and extract the asset id.
-   * The offer id might be the Cleeng format (`S<assetId>_<pricingOptionId>`) or the asset id as string.
-   */
-  private parseOfferId(offerId: string | number) {
-    if (typeof offerId === 'string') {
-      // offer id format `S<assetId>_<pricingOptionId>`
-      if (offerId.startsWith('C') || offerId.startsWith('S')) {
-        return parseInt(offerId.slice(1).split('_')[0]);
-      }
-
-      // offer id format `<assetId>`
-      return parseInt(offerId);
-    }
-
-    return offerId;
+  private formatOfferId(offer: PlanPrice) {
+    return `${offer.access.type === 'subscription' ? 'S' : 'C'}${offer.id}`; //`${offer.access.type === 'subscription' ? 'S' : 'C'}S${offer.item_id}_${offer.id}`;
```

Comment:
> [!WARNING]
> This change undos the fix from PR: https://github.com/jwplayer/ott-web-app/pull/541

Reply:
I'm aware of this, the fix was necessary for the previous implementation of offers with the JWP integration, but with the implementation introduced with this PR it's no longer needed.

Reply:
Do you mean that the previous integration can't/shouldn't be used anymore?

Reply:
No, I just reverted back to the `parseOfferId` and `formatOffer` methods the way they were done with your fix.